package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "7.0.0",
+  "version": "7.1.0",
```

Comment:
I'd leave it as 7.0.0.

Also, after updating it here, you should do `npm i` to automatically update the package lock file (which, right now, has the old version in it).

Reply:
Fixed.